actions.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+from PyQt5 import QtWidgets
+
+import streamlink
+from constants import MUTE_CHECKBOX
+from containers import LiveStreamContainer
+
+
+def mute_all_streams(parent):
```

Comment:
I don't know but these seems misplaced. Are we are splitting up the functionality of mutating the main window to this file? If so should not all mutators be placed here? Seems like an odd design choice imo.